tests/ensure_provider_tests.py

```diff
@@ -72,6 +72,7 @@
     "playbooks/tests_ipv6_disabled.yml": {
         EXTRA_RUN_CONDITION: "ansible_distribution_major_version == '8'",
     },
+    "playbooks/tests_ipv6_dns_search.yml": {},
```

Comment:
Why is this an NM-only test? Is setting DNS search not supported in initscripts?

Reply:
DNS search (`DOMAIN`) is supported in initscripts. But the situation is: "The role only set the DNS search setting for enabled IP family",  unlike NM, the `DOMAIN` is not distinguished on the IP  family, and `tests_ipv6_dns_search.yml` primarily test the specical behavior (The role only set the DNS search setting for enabled IP family) for NM backend.

But it is also true that we can test the DNS search setting for initscripts provider.